docs/UserGuide.md

```diff
@@ -3,71 +3,43 @@ layout: page
 title: User Guide
 ---
 
-Are you a CS student struggling to keep track of your internship?
-Do you feel tired of using spreadsheets to keep track of your applications?
-We have just the right tool for you!
+Are you a CS student struggling to keep track of your internship applications?
+Do you feel tired of using spreadsheets? We have just the right tool.
```

Comment:
Can probably keep 1-2 `!`, discuss on Thursday

Reply:
will update once we resolve on Thursday